src/api/Feed.js

```diff
@@ -439,6 +466,58 @@ class Feed extends Base {
         });
     }
 
+    /**
+     * Fetches replies (comments) of a comment or annotation
+     *
+     * @param {BoxItem} file - The file to which the comment or annotation belongs to
+     * @param {string} itemId - ID of the comment or annotation
+     * @param {CommentFeedItemType} itemType - Type of the comment or annotation
+     * @param {Function} successCallback
+     * @param {ErrorCallback} errorCallback
+     * @return {void}
+     */
+    fetchReplies(
+        file: BoxItem,
+        itemId: string,
+        itemType: CommentFeedItemType,
+        successCallback: (comments: Array<Comment>) => void,
+        errorCallback: ErrorCallback,
+    ): void {
+        const { id, permissions } = file;
+        if (!id || !permissions) {
+            throw getBadItemError();
+        }
+
+        this.file = file;
+        this.errorCallback = errorCallback;
+
+        this.updateFeedItem({ isRepliesLoading: true }, itemId);
+
+        const successCallbackFn = (comments: ThreadedCommentsType) => {
+            this.updateFeedItem({ isRepliesLoading: false, replies: comments.entries }, itemId);
+            successCallback(comments.entries);
+        };
+        const errorCallbackFn = (e: ErrorResponseData, code: string) => {
+            this.fetchRepliesErrorCallback(e, code, itemId);
+        };
+
+        if (itemType === 'annotation') {
```

Comment:
I think that's a very good idea, however I believe it should be done later as a separate change. Because it will affect many files, it might obscure the goal of this change. Also I think it needs to be done thoroughly to replace all instances of hardcoded types (all of them) and because of lack of time, should not be done right now.